test/functional/test_runner.py

```diff
@@ -224,7 +224,8 @@
     'wallet_importprunedfunds.py --descriptors',
     'p2p_leak_tx.py',
     'p2p_eviction.py',
-    'rpc_signmessage.py',
+    'rpc_signmessagewithaddress.py',
```

Comment:
wallet tests should to be prefixed with `wallet_` (see our docs on prefixes)

Reply:
Done, thanks for pointing out.